doc/source/SSHClient.rst

```diff
@@ -149,7 +151,7 @@ API: SSHClient and SSHAuth.
         :param verbose: Produce log.info records for command call and output
         :type verbose: ``bool``
         :param timeout: Timeout for command execution.
-        :type timeout: ``typing.Optional[int]``
+        :type timeout: ``typing.Union[int, None]``
```

Comment:
why? I remember previously you replaced `Union[x, None]` with `Optional`

Reply:
`None` is not default here, and Optional for cases, when None is default. (Anyway `Optional[T]` === `Union[None, T]`)

Reply:
I see